Integrations/CarbonBlackProtect/CarbonBlackProtect.py

```diff
@@ -0,0 +1,1105 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+''' IMPORTS '''
+
+import json
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+
+TOKEN = demisto.params().get('token')
+# Remove trailing slash to prevent wrong URL path to service
+SERVER = "{server}{api_endpoint}".format(
+    server=demisto.params()['url'][:-1] if (demisto.params()['url']
```

Comment:
I decided to not use an API_ENDPOINT const because I think it's unnecessary (it will be used just once, and doesn't make the code clearer), but I did add BASE_URL alongside SERVER to make easier to read